cs-vmware/src/main/java/io/cloudslang/content/vmware/services/ClusterComputeResourceService.java

```diff
@@ -114,7 +95,7 @@ public String getVmOverride(final HttpInputs httpInputs, final VmInputs vmInputs
 
             return restartPriority;
         } finally {
-            if(connectionResources != null && connectionResources.getConnection() != null) {
+            if (connectionResources != null && connectionResources.getConnection() != null) {
```

Comment:
Since we have closeSession input to the operation, can you also check if the user wants to close the session here too?